### ISSUE TITLE
Box UV sometimes creates UVs of 0 width.

### ISSUE THREAD

**Issue by jitspoe** [leukbaars/DreamUV#50]
Maybe best illustrated with a screenshot:

![image](https://github.com/user-attachments/assets/27e323de-c349-4cca-bcdf-84311810ef52)

I was just doing some quick tests and noticed this.  Not sure how often it would happen with "normal" geo, but figured I'd report it.

I fixed upper rightmost face by flattening it, but that solution didn't seem to work for others.

Attached is the blend file and texture to reproduce the issue:
[dreamuv_test_box_bug.zip](https://github.com/user-attachments/files/17337747/dreamuv_test_box_bug.zip)

This was in Blender 4.1.

**Comment by leukbaars** [comment]
thanks for providing a test case! I've seen it happen as well and have been meaning to fix this

**Comment by leukbaars** [comment]
rewrote the algorithm to something smarter, should work better now! Thanks again for providing test file

**Comment by jitspoe** [comment]
No, thank YOU for making such an awesome plugin and responding to bug reports!
